.travis.yml

```diff
@@ -84,7 +84,7 @@ script:
   - ./$EXE_NAME --help || test $? -eq 2
   - |
     if [ "$TRAVIS_OS_NAME" = linux ]; then
-      make controller.yaml controller-norbac.yaml CONTROLLER_IMAGE=$CONTROLLER_IMAGE
+      make controller.image controller.yaml controller-norbac.yaml CONTROLLER_IMAGE=$CONTROLLER_IMAGE
```

Comment:
`controller.image` target builds the Docker image.
It's not required for and breaks GoReleaser so the specific make target was added here.